dgi_image_discovery.module

```diff
@@ -5,10 +5,9 @@
  * General hook implementations.
  */
 
-use Drupal\dgi_image_discovery\DIDImageItemList;
-
 use Drupal\Core\Entity\EntityTypeInterface;
 use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\dgi_image_discovery\DIDImageItemList;
```

Comment:
Largely inconsequential, but should've been in the pre-existing coding standards commits; however, I spec'd the extensions as `php`, so it didn't touch the `.module`... whoops! Anyway.